keyboards/null/st110r2/info.json

```diff
@@ -0,0 +1,155 @@
+{
+    "manufacturer": "qmk",
+    "keyboard_name": "null/st110r2",
+    "url": "https://null-src.com/posts/nkbm-st110r2/post.php",
+    "maintainer": "user176176",
+    "bootloader": "halfkay",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "audio": false,
+        "midi": false,
+        "rgblight": false,
+        "backlight": false,
+        "unicode": false
+    },
+    "matrix_size": {"rows": 8, "cols": 15},
```

Comment:
`matrix_size` should be automatically calculated by the layout.. why do you need to specify it here?

Reply:
Good catch. I must have looked it up from that not data driven qmk fork on the product page that I used as the learning material for this.